pkg/runcfanotify/runcfanotify.go

```diff
@@ -416,6 +416,7 @@ func (n *RuncNotifier) watchPidFileIterate(pidFileDirNotify *fanotify.NotifyFD,
 	if err != nil {
 		return false, err
 	}
+	path = filepath.Join("/", os.Getenv("HOST_ROOT"), path)
```

Comment:
nit: We can use [`hostRoot`](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadgets/snapshot/process/tracer/tracer.go#L48) here as well.

Reply:
The hostRoot is in the snapshot process package whereas the runcfanotify package should be usable outside of IG. But I will reuse the same code pattern.